crates/rethnet_evm/src/block/builder.rs

```diff
@@ -165,6 +165,12 @@ where
                 AccountModifierFn::new(Box::new(move |balance, _nonce, _code| {
                     *balance += reward;
                 })),
+                &|| {
+                    Ok(AccountInfo {
+                        code: None,
+                        ..AccountInfo::default()
```

Comment:
What's (conceptually) the difference between `code: None` and `code: Some(Bytecode::new())`? Or, in other words, why don't we just do `Ok(AccountInfo::default())` here?

Reply:
The `code` field is intended to signal whether we're locally storing the code or - if `None` whether we're caching it elsewhere. We always store it separately, so I just want to make sure to adhere to this convention.

I am planning to propose a change to `revm` to make the separation clearer using an enum:

```rust
pub enum CodeRef {
  Local(Bytecode),
  Ref(B256),
}
```

that way we also avoid the duplication of having a stored `code_hash` when it's also stored in the `Bytecode`

Reply:
Makes sense, thanks!